### PR TITLE
Merge branch 2.0 into branch 2.1

### PR DESCRIPTION
**Pull request by damien-carcel**
## Definition Of Done

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | -
| Added Behats                      | -
| Added integration tests           | -
| Changelog updated                 | -
| Review and 2 GTM                  | OK
| Micro Demo to the PO (Story only) | -
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: CHANGELOG-2.0.md [hunk 1]
```diff
@@ -1,4 +1,12 @@
-# 2.0.x
+# 2.0.9 (2017-12-15)
+
+## Bug fixes
+
+- PIM-7037: Allows code to be an integer on product model import
+- PIM-7011: XLS Options Import - Simple select attribute option cannot be updated for options with numeric codes
```
Comment (pierallard) [inline CHANGELOG-2.0.md]: Not related, but should be XLSX, no?

Comment (damien-carcel) [reply CHANGELOG-2.0.md]: Yes it should. Like for an issue on 1.7 change log, I'll change it in 2.0.

File: src/Pim/Bundle/EnrichBundle/Resources/public/js/family/form/variants/add-family-variant-fields-container.js [hunk 2]
```diff
@@ -198,14 +200,16 @@ define(
                 $.when.apply($, fetchAttributesPromises)
                     .then(function () {
                         const data = _.map(arguments, (attribute) => {
-                            return {
+                            const label = this.getEntityLabel(attribute, catalogLocale);
+
+return {
```
Comment (pierallard) [inline src/Pim/Bundle/EnrichBundle/Resources/public/js/family/form/variants/add-family-variant-fields-container.js]: Really weird

Comment (damien-carcel) [reply src/Pim/Bundle/EnrichBundle/Resources/public/js/family/form/variants/add-family-variant-fields-container.js]: Yes. But the problem is in 2.0, not in this merge. I'll fix it in 2.0, we'll do very soon (today if possible) a second 2.0→2.1 merge to bring the Symfony 3.4 update.

**Pull request merged**